assets/src/edit-story/components/form/radioGroup.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+
+const RadioButton = styled.div`
+  min-height: 30px;
+  margin: 10px 0;
+`;
+
+const Radio = styled.input`
+  opacity: 0;
+  position: absolute;
+  :focus + label {
+    outline: -webkit-focus-ring-color auto 5px;
+  }
+`;
+
+const RADIO_SIZE = 20;
+const DOT_SIZE = 10;
+const TEXT_OFFSET = 30;
+
+const Label = styled.label`
```

Comment:
This doesn't look correct to me - the dot is off-center:

![Screen Shot 2020-03-23 at 3 36 15 PM](https://user-images.githubusercontent.com/637548/77356557-ee22ee80-6d1c-11ea-9083-0a0923508024.png)

Reply:
Are you by any chance zoomed in/out when the dot is off-center. I couldn't see the issue until I zoomed in and now can see this.

I suppose using an SVG would solve the issue.

Reply:
Using SVGs now.